boards/arm/96b_carbon/doc/96b_carbon.rst

```diff
@@ -10,6 +10,39 @@ Zephyr applications use the 96b_carbon configuration to run on the 96Boards
 Carbon hardware. It is based on the STMicroelectronics STM32F401RET Cortex-M4
 CPU and also contains a nRF51832 chip connected over SPI for BLE connectivity.
 
+.. note::
```

Comment:
I'm leaning more toward's @pfalcon's comment on this one.  I also don't think a NOTE should be this long either.  So here's a suggested rewrite of this section (all the way down to the figure):

     The 96b_carbon board is built with two chips: an STMicroelectronics
     STM32F401RET Cortex-M4 CPU and an nRF51832 chip connected to
     the Cortex-M4 CPU over SPI for Bluetooth LE connectivity.  Even though
     both chips exist on the same physical board, they must be programmed
     separately:

     - The ``96b_carbon`` configuration is used when developing programs for
       the main chip on the board, the STM32F401RET. Users will likely want to
       write applications targeting this chip, using the ``96b_carbon``
       configuration, since it is connected to all of the breakout
       I/O headers.

     - The ``96b_carbon_nrf51`` configuration should be used for programming
       the secondary nRF51832 chip. Most users will likely not develop
       applications for this chip, since Zephyr already provides a 
       sample application that can be flashed onto the nRF51832
       to provide Bluetooth functionality to applications on the main
       STM32F401RET chip.

     For instructions on how to set up the nRF51832 to develop Bluetooth
     applications, see :ref:`96b_carbon_nrf51_bluetooth`.
 
     After you have flashed your nRF51, you can perform basic validation
     of this Bluetooth setup using the instructions
     :ref:`below<96b_carbon_verify_bluetooth>`.